oio/cli/admin/service_check.py

```diff
@@ -255,3 +257,54 @@ def take_action(self, parsed_args):
         self.logger.info("All rdir services are alive.")
 
         return ('Status', ), [('Ok', )]
+
+
+class RawxCheck(ItemCheckCommand):
+    """
+    Check all rawx chunks
+    """
+
+    columns = ('Chunk', 'Status', 'Errors')
+
+    @property
+    def rdir(self):
+        """Get an instance of RdirClient."""
+        return self.app.client_manager.rdir
+
+    reqid_prefix = 'ACLI-RC-'
+
+    def _format_results(self, checker):
+        for res in checker.run():
+            if res.target.type == "chunk":
+                yield (repr(res.target)[len('chunk='):],
+                       res.health,
+                       res.errors_to_str())
+
+    def get_parser(self, prog_name):
+        parser = super(RawxCheck, self).get_parser(prog_name)
+        parser.add_argument(
+            'services',
+            nargs='*',
+            metavar='<services>',
+            help=('Path of the volume to check.'
+                  'Disclaimer: This only works on the same machine')
+        )
+        return parser
+
+    def _prefix_http(self, url):
+        if not url.startswith('http://'):
+            return "http://" + url
+
+    def take_action(self, parsed_args):
+        super(RawxCheck, self).take_action(parsed_args)
+        reqid = self.app.request_id(self.reqid_prefix)
+        checker = self.build_checker(parsed_args)
+        for service in parsed_args.services:
+            chunks = [x for x in
+                      self.rdir.chunk_fetch(service, limit=1000, reqid=reqid)]
```

Comment:
Please stop building huge lists in CLI code. When doing this, the output is frozen, and user does not understand what is happening. Use generators.